astronomer/providers/apache/hive/hooks/hive.py

```diff
@@ -0,0 +1,27 @@
+"""This module contains the Apache HiveCli hook async."""
+from airflow.hooks.base import BaseHook
+from impala.dbapi import connect
+from impala.hiveserver2 import HiveServer2Connection
+
+
+class HiveCliHookAsync(BaseHook):
+    """
+    Wrapper to interact with the Hive using impyla library
+
+    :param conn_id: connection string for the hive
```

Comment:
We shall mention the default connection name here if it we have

Reply:
made it to default metastore_conn_id